src/main/java/bio/terra/stairway/FlightContext.java

```diff
@@ -14,8 +14,8 @@
     private FlightMap workingMap; // open-ended state used by the steps
     private int stepIndex; // what step we are on
     private boolean doing; // true - executing do's; false - executing undo's
-    private StepResult result; // current status
-
+    private StepResult result; // current step status
+    private FlightStatus flightStatus; // flight complete status
```

Comment:
this is the actual status, not just the completed status, correct?

Reply:
Yes, I'll clarify the comment.